src/main/java/Objects/Packets/Botnet.java

```diff
@@ -1,11 +1,20 @@
 package Objects.Packets;
 
+/**
+ * A specialized packet used to take control
+ * of other devices.
+ */
 public class Botnet extends Packet {
 
-    public Botnet(final int team) {
-        //super(team);
-        /*setCost(1);
-        setStealth(0);*/
+    /**
+     * Constructor for the Botnet packet type.
+     * @param team Team that owns the packet.
+     */
+    public Botnet(final String team) {
```

Comment:
Same as with NetworkDevice, why not a general packet constructor to handle team?

Reply:
Good idea.
Fixing now.

Reply:
unchanged